python/cuml/cluster/hdbscan.pyx

```diff
@@ -493,6 +525,7 @@ class HDBSCAN(Base, ClusterMixin, CMajorInputTagMixin):
 
         self.fit_called_ = False
         self.prediction_data = prediction_data
+        self.prediction_data = prediction_data
```

Comment:
Is this additional line from a bad merge? 

Related but off-topic(-ish), the reference impl stores the resulting prediction data object on the attribute name `prediction_data_`. I would prefer to add something else to the name we use so that it doesn't confuse users- maybe something like `self.prediction_data_ptr` could help (I haven't checked so disregard if you've already done this).